src/codeu/chat/common/User.java

```diff
@@ -62,22 +64,41 @@ public User read(InputStream in) throws IOException {
           Serializers.STRING.read(in),
           Time.SERIALIZER.read(in)
       );
-
     }
   };
   
   public HashMap<Uuid, Time> UserUpdateMap = new HashMap<Uuid, Time>();
   public HashMap<Uuid, Time> ConvoUpdateMap = new HashMap<Uuid, Time>();
 
+  public void add(Uuid conversation, Access access) {
+      map.put(conversation, access);
+  }
+
+  public void remove(Uuid conversation) {
+    map.remove(conversation);
+  }
+
+  public boolean containsConversation(Uuid conversation) {
+    return map.containsKey(conversation);
+  }
+
+  public Access get(Uuid conversation) {
+    return map.get(conversation);
+  }
+
   public final Uuid id;
   public final String name;
   public final Time creation;
 
+  // map from conversation id to Access
+  private Map<Uuid, Access> map;
```

Comment:
give this a more descriptive name, and move it closer to the other instance variables.